tests/integ/test_mxnet_train.py

```diff
@@ -109,6 +110,30 @@ def test_deploy_model_with_tags(mxnet_training_job, sagemaker_session, mxnet_ful
         assert production_variants[0]['InitialInstanceCount'] == 1
 
 
+def test_deploy_model_with_kms_key(mxnet_training_job, sagemaker_session, mxnet_full_version):
+    endpoint_name = 'test-mxnet-deploy-model-{}'.format(sagemaker_timestamp())
+
+    with timeout_and_delete_endpoint_by_name(endpoint_name, sagemaker_session):
+        desc = sagemaker_session.sagemaker_client.describe_training_job(TrainingJobName=mxnet_training_job)
+        model_data = desc['ModelArtifacts']['S3ModelArtifacts']
+        script_path = os.path.join(DATA_DIR, 'mxnet_mnist', 'mnist.py')
+        model = MXNetModel(model_data, 'SageMakerRole', entry_point=script_path,
+                           py_version=PYTHON_VERSION, sagemaker_session=sagemaker_session,
+                           framework_version=mxnet_full_version)
+
+        sts_client = sagemaker_session.boto_session.client('sts')
+        account_id = sts_client.get_caller_identity()['Account']
+        kms_client = sagemaker_session.boto_session.client('kms')
+        kms_key_arn = get_or_create_kms_key(kms_client, account_id)
```

Comment:
Optional - I think it will make this code cleaner if get_or_create_kms_key just take sageamker_session as the argument.

Reply:
done